modules/vba_documents/spec/sidekiq/upload_upstream_processing_error_batch_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe VBADocuments::UploadUpstreamProcessingErrorBatch, type: :job do
+  let(:client_stub) { instance_double(CentralMail::Service) }
+
+  # test target, less than 14 days old in emms error status, job should pick up this record
+  let!(:emms_error_upload) { create(:upload_submission, :status_emms_error) }
+
+  # Old errored record, job should not pick up since it's older than 14 days
+  let!(:old_error_upload) { create(:upload_submission, :status_emms_error, created_at: 20.days.ago) }
+
+  # DOC104 error, error not from EMMS, job should not pick up
+  let!(:error_upload) { create(:upload_submission, :status_error) }
+
+  # No error, job should not pickup
+  let!(:no_error_upload) { create(:upload_submission) }
+
+  let(:faraday_response) { instance_double(Faraday::Response) }
+  let(:in_process_element) do
+    [{ uuid: 'ignored',
+       status: 'VBMS Complete',
+       errorMessage: '',
+       lastUpdated: '2018-04-25 00:02:39' }]
+  end
```

Comment:
What does `:in_process_element` mean here? This looks like a response from the upstream service stating that the submission is complete. Should it be something like `upstream_service_response_body` or `central_mail_response_body` instead?